rocketpool-cli/pdao/get-settings.go

```diff
@@ -68,9 +68,9 @@ func getSettings(c *cli.Context) error {
 	fmt.Printf("\tNode Penalty Quorum:        %.2f%%\n", response.Network.NodePenaltyThreshold*100)
 	fmt.Printf("\tPenalty Size:               %.2f%%\n", response.Network.PerPenaltyRate*100)
 	fmt.Printf("\tBalance Submission Enabled: %t\n", response.Network.IsSubmitBalancesEnabled)
-	fmt.Printf("\tBalance Submission Freq:    %d Blocks\n", response.Network.SubmitBalancesFrequency)
+	fmt.Printf("\tBalance Submission Freq:    %d Seconds \n", response.Network.SubmitBalancesFrequency)
```

Comment:
Same comment, if we treat this as a duration we can print it in human-readable format instead of a raw number of seconds